test/fixtures/jsx/index.js

```diff
@@ -1,11 +1,15 @@
 const h = (tag, props, ...children) => ({ tag, props, children });
+const Fragment = ({ children }) => children;
 
 export default class Foo {
 	render() {
 		return (
 			<div id="app">
 				<h1>Hello, World!</h1>
 				<p>A JSX demo.</p>
+				<Fragment>
```

Comment:
seems like I can't get `<>` to work with pragmaFrag

Reply:
that's weird, even with it included at the top?

Reply:
It seemed like flow was the issue, moving to babel-flow-preset fixed the issue.